wordpress-mobile/WordPress-iOS
# Comment Detail: Content cell initial view

Refs #17108, #17087
Depends on #17106

This is the first part of #17108, which adds the initial table view cell for comment content. Note that the cell is not yet populated by a Comment object, and therefore looks empty.

Some preview:

![Simulator Screen Shot - iPhone 8 - 2021-09-02 at 19 38 05](https://user-images.githubusercontent.com/1299411/131844979-3bb9547f-3f58-47f1-89b1-dfbca4dd4313.png)

Notes:
- I'm using SF Symbols for the button icons, since they are available to use on iOS 13.0.
- I noticed that the buttons (accessory, reply, like) have no highlighted state. I'll add them in a separate PR.
- In the next iteration, I'll populate the cell with a `Comment` object.

## To test

- Ensure the `New Comment Detail` feature flag is enabled.
- Go to My Site > Comments, and tap any comment.
- Verify that the comment content cell is displayed correctly:
  - The author's avatar image is displayed correctly.
  - The author's name is displayed correctly.
  - The date of the comment is displayed correctly.
  - The number of likes is displayed correctly.
  - The state of the liked button is correctly set (blue if liked, hollow with `.textSubtle` if unliked). Note that the like/unlike toggle is not implemented yet, so this is expected.

## Regression Notes
1. Potential unintended areas of impact
n/a. Feature is hidden behind a feature flag.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a. Feature is hidden behind a feature flag.

3. What automated tests I added (or what prevented me from doing so)
n/a. Feature is hidden behind a feature flag.

PR submission checklist:
